library/Transformers/DeprecatedKeyValue.php

```diff
@@ -50,15 +49,13 @@ static function ($input) use ($comparedKey, $ruleName, $baseKey) {
                     try {
                         return new Key($comparedKey, Validator::__callStatic($ruleName, [$input[$baseKey]]));
                     } catch (Throwable) {
-                        $rule = new AlwaysInvalid();
-                        $rule->setName($comparedKey);
-                        $rule->setTemplate(sprintf(
-                            '%s must be valid to validate %s',
-                            $baseKey,
-                            $comparedKey,
-                        ));
-
-                        return $rule;
+                        return (new AlwaysInvalid())
+                            ->evaluate($baseKey)
+                            ->withPath($baseKey)
+                            ->withTemplate('{{name}} must be valid to validate {{comparedKey}}')
+                            ->withExtraParameters([
+                                'comparedKey' => $comparedKey,
```

Comment:
```suggestion
                                'comparedKey' => new Path($comparedKey),
```

Once we have our own stringifier, that would be doable